lib/src/app.dart

```diff
@@ -1,85 +1,65 @@
-          // background.
-          restorationScopeId: 'app',
-
-          // Provide the generated AppLocalizations to the MaterialApp. This
-          // allows descendant Widgets to display the correct translations
-          // depending on the user's locale.
-          localizationsDelegates: const [
-            AppLocalizations.delegate,
-            GlobalMaterialLocalizations.delegate,
-            GlobalWidgetsLocalizations.delegate,
-            GlobalCupertinoLocalizations.delegate,
-          ],
-          supportedLocales: const [
-            Locale('en', ''), // English, no country code
-          ],
-
-          // Use AppLocalizations to configure the correct application title
-          // depending on the user's locale.
-          //
-          // The appTitle is defined in .arb files found in the localization
-          // directory.
-          onGenerateTitle: (BuildContext context) =>
-              AppLocalizations.of(context)!.appTitle,
-
-          // Define a light and dark color theme. Then, read the user's
-          // preferred ThemeMode (light, dark, or system default) from the
-          // SettingsController to display the correct theme.
-          theme: ThemeData(),
-          darkTheme: ThemeData.dark(),
-          themeMode: settingsController.themeMode,
-
-          // Define a function to handle named routes in order to support
-          // Flutter web url navigation and deep linking.
-          onGenerateRoute: (RouteSettings routeSettings) {
-            return MaterialPageRoute<void>(
-              settings: routeSettings,
-              builder: (BuildContext context) {
-                switch (routeSettings.name) {
-                  case SettingsView.routeName:
-                    return SettingsView(controller: settingsController);
-                  case SampleItemDetailsView.routeName:
-                    return const SampleItemDetailsView();
-                  case SampleItemListView.routeName:
-                  default:
-                    return const SampleItemListView();
-                }
-              },
-            );
-          },
-        );
-      },
+    return DefaultTabController(
+      length: 3,
+      child: MaterialApp(
+        debugShowCheckedModeBanner: false,
+        home: Scaffold(
+          drawer: NavigationDrawerWidget(),
+          appBar: AppBar(
+            bottom: const TabBar(
+              indicatorColor: Colors.white,
+              tabs: [
+                Tab(icon: Icon(Icons.chat_bubble_rounded), text: 'Chat'),
+                Tab(icon: Icon(Icons.person), text: 'Request'),
+                Tab(icon: Icon(Icons.phone), text: 'Calls')
+              ],
+            ),
+            backgroundColor: const Color(0xffD6521B),
+            centerTitle: true,
+            title: const Text('Tutor Me'),
+            actions: const <Widget>[],
+          ),
+          body: Column(
+            children: <Widget>[
+              for (var i in people)
```

Comment:
info • The value of the local variable 'i' isn't used • lib/src/app.dart:39:24 • unused_local_variable

@MadunaThabo @MadunaThabo